MAUI/Kanban-Board/Cards.md

```diff
@@ -42,83 +42,83 @@ You can replace the entire card template with your own design using [`CardTempla
 
 {% highlight xaml %}
 
-    <kanban:SfKanban.CardTemplate >
+<kanban:SfKanban.CardTemplate >
 
-    <DataTemplate>
-        <StackLayout WidthRequest="250" Orientation="Vertical" BackgroundColor="Gray" Padding="10,10,10,10"> 
-            
-            <StackLayout  Orientation="Horizontal"> 
+<DataTemplate>
+    <StackLayout WidthRequest="250" Orientation="Vertical" BackgroundColor="Gray" Padding="10,10,10,10"> 
+        
+        <StackLayout  Orientation="Horizontal"> 
 
-                <Label Text="{Binding Path=Title}" TextColor="Silver" HorizontalOptions="StartAndExpand" >
-                </Label>
+            <Label Text="{Binding Path=Title}" TextColor="Silver" HorizontalOptions="StartAndExpand" >
+            </Label>
 
-            </StackLayout>      
+        </StackLayout>      
 
-            <StackLayout  Orientation="Horizontal"> 
+        <StackLayout  Orientation="Horizontal"> 
 
-                <Label Text="{Binding Description}" WidthRequest="150" FontSize="14" TextColor="Silver" LineBreakMode="WordWrap" ></Label>                    
-                <Image Source="{Binding ImageURL}" HeightRequest="50" WidthRequest="50"  ></Image>
+            <Label Text="{Binding Description}" WidthRequest="150" FontSize="14" TextColor="Silver" LineBreakMode="WordWrap" ></Label>                    
+            <Image Source="{Binding ImageURL}" HeightRequest="50" WidthRequest="50"  ></Image>
 
-            </StackLayout>
-            
         </StackLayout>
-    </DataTemplate>
+        
+    </StackLayout>
+</DataTemplate>
 
-    </kanban:SfKanban.CardTemplate>
+</kanban:SfKanban.CardTemplate>
 
 {% endhighlight %}
 
 {% highlight C# %}
 
-    var cardTemplate = new DataTemplate(() =>
+var cardTemplate = new DataTemplate(() =>
+{
+
+    StackLayout root = new StackLayout()
+    {
+        WidthRequest = 250,
+        Orientation = StackOrientation.Vertical,
+        Padding = new Thickness(10),
+        BackgroundColor = Color.Gray
+    };
+
+    StackLayout titleLayout = new StackLayout();
+    Label title = new Label()
+    {
+        TextColor = Color.Silver,
```

Comment:
@EzhilarasanElangovan31 in maui, we do not have Color.Silver, instead use Colors.Silver

Reply:
Updated